src/components/Quotes/Quotes.js

```diff
@@ -1,33 +1,34 @@
 import React from 'react';
 import styles from './Quote.css';
-import { useQuotes } from '../../hooks/quotes';
+import { useQuotes, searchQuotes } from '../../hooks/quotes';
 
 const Quotes = () => {
-  const { handleClick, handleNumQuotesChange, searchQuotes, numQuotes, quotes } = useQuotes();
+  const { handleClick, handleNumQuotesChange, numQuotes, quotes } = useQuotes();
 
   return (
     <>
       <div className={styles.Quote}>
         <button onClick={() => handleClick('https://ron-swanson-quotes.herokuapp.com/v2/quotes/1')}>Get Random Quote</button>
-        <div>
-          <input onChange={({ target }) => searchQuotes(target.value)} name="searchTerm" >
-            <button onClick={() => handleClick(`https://ron-swanson-quotes.herokuapp.com/v2/quotes/${searchTerm}/1`)}>Search Quotes</button>
-          </input>
-        </div>
-        <div>
-          <button onClick={() => handleClick(`https://ron-swanson-quotes.herokuapp.com/v2/quotes/${numQuotes}`)}>Get {numQuotes} Quotes</button>
-          <input className='numQuotes' onChange={({ target }) => handleNumQuotesChange(parseInt(target.value))} type='range' min='2' max='100' value={numQuotes}></input>
-        </div>
-
+      </div>
+      <div>
+        <input type="text" name="searchTerm" >
+        </input>
+        <button onClick={({ target }) => searchQuotes(target.value)}>Search Quotes</button>
+      </div>
+      <div>
+        <button onClick={() => handleClick(`https://ron-swanson-quotes.herokuapp.com/v2/quotes/${numQuotes}`)}>Get {numQuotes} Quotes</button>
+        <input className='numQuotes' onChange={({ target }) => handleNumQuotesChange(parseInt(target.value))} type='range' min='2' max='100' value={numQuotes}></input>
+      </div>
 
+      <ul>
         { quotes.length ?
```

Comment:
This won't work. The API you are using returns an array of 1 or more quotes, not an array of objects. You can't destructure text, name , or id from your array because they don't exist.